native/cocos/audio/apple/AudioEngine-inl.mm

```diff
@@ -96,7 +96,7 @@ - (id)init {
         }
 
         BOOL success = [[AVAudioSession sharedInstance]
-            setCategory:AVAudioSessionCategoryPlayback
+            setCategory:AVAudioSessionCategoryAmbient
```

Comment:
Basis of repair:
https://developer.apple.com/documentation/avfaudio/avaudiosession/category
![image](https://github.com/cocos/cocos-engine/assets/3078423/75786e2a-baba-4b9b-b79c-600ac7cf20c9)

Reply:
This API supports iOS 12.0+, but cocos supports iOS 11+. So need to keep compatible.